src/utils/HttpClient.js

```diff
@@ -1,66 +0,0 @@
-import request from 'superagent';
-import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
-
-function getUrl(path) {
```

Comment:
So here's the rundown:

create axios clients on both client and server, setup the interceptors, then pass each instances to the `configureStore` , and finally create promise middleware that will accept the different clients...and we're done.